linode/manager
# feat: [M3-6787] - Update LKE HA implementation

## Description 📝
Replaces the former HA Checkbox with a Radio Group and moves the HA Control Plane away from the KubeCheckoutBar into the main content.

## Major Changes 🔄
- New `HAControlPlane` component consisting of a `RadioGroup` forcing users to make an active selection for HA
- Remove `HACheckbox` component from the `KubeCheckoutBar` component in the sidebar.
- User's choice to either opt-in or opt-out of HA is now required to create a Cluster.

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![beforeChanges](https://github.com/linode/manager/assets/119514965/6b92eb5d-1f9b-4503-96bb-53818f0e7613) | ![afterChanges](https://github.com/linode/manager/assets/119514965/a6165fd5-9b12-44ca-8a61-3e64d549daaa) |



## How to test 🧪
1. Visit the `kubernetes/create` landing page and inspect for any UI defects.
2. Test the functionality of the new `HAControlPlane` component.
3. Verify that LKE HA flow works as expected.
